src/lang/locales.js

```diff
@@ -0,0 +1,11 @@
+import i18n from 'i18n';
+
+export default function initLocale() {
+  i18n.configure({
+    cookie: 'lang',
+    defaultLocale: 'en',
+    directory: `${__dirname}/locales`,
+    locales: ['en', 'ru'],
+    register: global,
```

Comment:
Agreed. Have changed the method's name to be 'locale'. Speaking about password, I moved it to a different table/controller because I don't feel like storing it with user's info. I'd rather divide a logic of storing user's info (name, birthday and so on that is public info) and his credentials (that should be private and secured).

Reply:
Yeah, but it's an opportunity to implement filtering sensitive data before sending it to the client.
It could be even more interesting if we have roles in the application and different filtering depending on user role.